README.rst

```diff
@@ -104,12 +104,13 @@ packaged in "providers".
 .. code:: python
 
     from faker import Faker
-    from faker.providers import internet
 
     fake = Faker()
-    fake.add_provider(internet)
 
     print(fake.ipv4_private())
+    print(fake.address())
+
+**Note:** The `add_provider`_ method is not necessary unless you are using external providers - the bundled providers are already available out of the box.
```

Comment:
To me it makes it a little puzzling to mention the `The add_provider method` in this paragraph, when it's not in the code example above.